### ISSUE TITLE
Language Request: COBOL

### ISSUE THREAD

**Issue by Pacatro** [o2sh/onefetch#1443]
### Language Name 🖊

COBOL

### Logo 📷

![programming-language-cobol-icon-2048x1761-bxw1kew6](https://github.com/user-attachments/assets/e7018ad6-7116-4f98-8444-c43d2251cbd5)

### Upstream support ✅

- [X] Does tokei already support the language in question?

**Comment by spenserblack** [comment]
Frankly, I'm surprised we don't already support it :sweat_smile:

If you're planning on making a PR I'd suggest waiting until #1305 drops, which might happen soon.

**Comment by Pacatro** [comment]
> Frankly, I'm surprised we don't already support it :sweat_smile:
> 
> If you're planning on making a PR I'd suggest waiting until #1305 drops, which might happen soon.

Got It! I'll keep an eye on that 😁

**Comment by Pacatro** [comment]
Hi again! Just wondering if it’s cool to go ahead with the PR now.

**Comment by spenserblack** [comment]
Yeah, I think you can go ahead and make a PR. In hindsight, there wasn't really a reason to wait.
